src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -64,8 +65,34 @@
       (.get spreadsheet-id)
       (.execute)))
 
+(defn get-sheet-titles
+  [service spreadsheet-id]
+  "returns a list of [sheet-title sheet-id] tuples. It seems the order reflects
+  the order of the tabs in google's interface, though I doubt this is anywhere
+  guaranteed."
+  (->> (get (get-sheet-info service spreadsheet-id) "sheets")
+       (map #(get % "properties"))
+       (map (juxt #(get % "title") #(get % "sheetId")))))
+
+(defn date-time
+  "using the given excel date formatting string, eg \"yyyy-mm-dd\", \"M/d/yyyy\" "
+  [date-time format-patterm]
```

Comment:
Maybe there's a protocol that wants to be expressed here? `Cellable` or some such, with `to-cell` and `from-cell` fns?

It's a little weird that the format-pattern applies on a cell and not on a column.